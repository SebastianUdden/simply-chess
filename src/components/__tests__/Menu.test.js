import React from "react"
import renderer from "react-test-renderer"
import Menu from "../Menu"

describe("Menu", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Menu score={{ white: 0, black: 0 }} onClick={() => {}} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
