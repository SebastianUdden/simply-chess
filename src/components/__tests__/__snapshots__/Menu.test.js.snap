// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<div
  className="sc-AxjAm lcVokx"
>
  <p
    className="sc-AxiKw dsXsSA"
  >
    <strong
      className="sc-AxhCb cEzvRj"
      id="score-white"
    >
      0
    </strong>
     -
     
    <strong
      className="sc-AxhUy tHwye"
      id="score-black"
    >
      0
    </strong>
  </p>
  <button
    className="sc-AxirZ kLfxfB"
    id="play"
    onClick={[Function]}
  >
    Play
  </button>
</div>
`;
